### ISSUE TITLE
WEBGL_USE_SHAPES_UNIFORMS causes test failures on MacOS

### ISSUE THREAD

**Issue by mattsoulanille** [tensorflow/tfjs#5479]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow.js):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): MacOS 10.13.6
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow.js installed from (npm or script link):
- TensorFlow.js version (use command below):
- Browser version: Chrome 92.0.4515.107
- Tensorflow.js Converter Version:

**Describe the current behavior**
WebGL tests using `WEBGL_USE_SHAPES_UNIFORMS` fail on MacOS Chrome 92.
**Describe the expected behavior**
WebGL tests using `WEBGL_USE_SHAPES_UNIFORMS` pass on MacOS Chrome 92.

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/CodePen/any notebook.

`yarn run-browserstack --browsers=bs_chrome_mac --testEnv webgl2 --flags '{"WEBGL_USE_SHAPES_UNIFORMS": true}'`

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

```
Chrome 92.0.4515.107 (Mac OS 10.13.6) where webgl2 {"WEBGL_USE_SHAPES_UNIFORMS":true} Tensor3D FAILED
	Error: Arrays differ: actual[2] = 3, expected[2] = 5.
```
[More logs in this nightly run](https://console.cloud.google.com/cloud-build/builds/b634d7e6-4b93-46b4-b484-1a5c5f99d578?project=834911136599)

**Comment by qjia7** [comment]
I will take a look why there are so many failures with 
WEBGL_USE_SHAPES_UNIFORMS = true.

**Comment by mattsoulanille** [comment]
Fixed by #5502  (not sure why the automation didn't close it).
